perun-core/src/main/java/cz/metacentrum/perun/core/entry/AttributesManagerEntry.java

```diff
@@ -4522,6 +4522,18 @@ public void convertAttributeToUnique(PerunSession session, int attrId) throws Pr
 		getAttributesManagerBl().convertAttributeToUnique(session, attrId);
 	}
 
+	@Override
+	public void convertAttributeToNonunique(PerunSession session, int attrId) throws PrivilegeException, AttributeNotExistsException {
+		Utils.checkPerunSession(session);
+
+		// Authorization
+		if(!AuthzResolver.authorizedInternal(session, "convertAttributeToNonunique_int_policy")) {
+			throw new PrivilegeException("This operation can do only PerunAdmin.");
```

Comment:
For this exception, I would not use this kind of specific message, because you can't be sure what is really set in the policy and if anyone changes the policy, this text would be misleading. Use more generic message like:
```
throw new PrivilegeException(sess, "convertAttributeToNonunique");
```

Reply:
Ok, fixed (even in convertAttributeToUnique).